client/components/CandidateReview/CandidateModals/NotApprovedModal/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import BasicModal from '../../../common/BasicModal';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCheck } from '@fortawesome/free-solid-svg-icons';
+import PropTypes from 'prop-types';
+import '../common.css';
+
+const NotApprovedModal = ({ handleAction = () => {}, githubUrl = '#' }) => {
+  return (
+    <BasicModal
+      show={true}
+      centered={true}
+      closeButton={false}
+      content={
+        <>
+          <p className="review-confirmation-content">Thank you for reviewing</p>
+          <p className="review-confirmation-share">
+            if you haven’t opened any issues yet, please{' '}
+            <a href={githubUrl} target="_blank" rel="noreferrer">
+              open an issue describing why this is not approved.
+            </a>
+          </p>
+        </>
+      }
+      dialogClassName="review-confirmation"
+      actions={[
+        {
+          label: 'Close',
+          onClick: handleAction
+        }
+      ]}
+      title={
+        <div className="review-confirmation-title">
+          <FontAwesomeIcon
+            icon={faCheck}
```

Comment:
Thinking an orange info circle or so could quickly reflect that this is the "non-final" path. I still thought I had incorrectly selected "Approved" instead of "Not Approved" at the end.

Just a thought, what do you think?

Reply:
The green check felt odd to me too but I wasn't confident enough about what sentiment we wanted to convey about 'not approved' reviews. Now that I know it feels odd to you as well, I'll swap it out.

Reply:
This was challenging. I decided that the exclamation triangle felt most accurate after trying a few others including the info circle. That change is [in this commit](https://github.com/w3c/aria-at-app/pull/1206/commits/201e8208153675290b83c85b127c081d04281990) but I don't feel attached to that decision. That particular "i" just felt off for a modal that didn't have information. I felt that the exclamation mark conveyed the need for attention without being too severe since it is orange. Another option is a pause to indicate the process is on hold or waiting further approval. Here are some screenshots. Do you have a preference?

<img width="405" alt="modal with orange info i in circle" src="https://github.com/user-attachments/assets/58606f59-86d6-4397-b899-45823005a318">
<img width="405" alt="modal with orange triangle exclamation" src="https://github.com/user-attachments/assets/5c93bd14-5f31-4a25-a9f0-05e88c0af16c">
<img width="453" alt="modal with info i with no circle" src="https://github.com/user-attachments/assets/b0c42227-e684-4cdb-afc6-fea4c5ceb9df">

Reply:
I'm also not strongly tied to either (except the 'i' with no background). Moving forward with the triangle seems fine to me! But thanks so much for the thought you put into that